src/System.Private.CoreLib/shared/System/Threading/WaitHandle.cs

```diff
@@ -137,7 +138,7 @@ private bool WaitOneNoCheck(int millisecondsTimeout)
 
             // The field value is modifiable via the public <see cref="WaitHandle.SafeWaitHandle"/> property, save it locally
             // to ensure that one instance is used in all places in this method
-            SafeWaitHandle waitHandle = _waitHandle;
+            SafeWaitHandle? waitHandle = _waitHandle;
```

Comment:
we could've changed this to use `??` as you did latter on this file.

Reply:
Will change.